packages/path-provider/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.2.0] - 2017-05-10
+
+* Upgrade to new plugin registration.
```

Comment:
Maybe add a link to https://groups.google.com/forum/#!topic/flutter-dev/zba1Ynf2OKM ?